File: poo-js-com-ts/src/poo_pessoa/Pessoa.ts
export class Pessoa {
  private nome: string;
  private idade: number;
  private id: number;

  public constructor(nome: string, idade: number, id: number) {
  
    this.nome = nome;
    this.idade = idade;
    this.id = id;
  
  }

  public getNome() : string {
    return this.nome;
  }

  public setNome(nome:string) : void {
    this.nome = nome;
  } 

  public getIdade() : number {
    return this.idade;
  }

  public setIdade(idade:number) : void {
    this.idade = idade;
  }

  public getId() : number {
    return this.id;
  }

  public setId(id:number) : void {
    this.id = id;
  }

  public toString() : string {
    return `nome: ${this.nome}\nidade: ${this.idade}\nid: ${this.id}`;
  }
}

File: exercicios-java/src/poo_pessoa/Pessoa.java
package poo_pessoa;

public class Pessoa {
	private String nome;
	private int idade;
	private int id; // RG
	
	public Pessoa() {
		nome = "-";
		idade = 0;
	    id = 0;
	}
	
	public Pessoa(String nome, int idade, int id) {
		this.nome = nome;
		this.idade = idade;
		this.id = id;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public int getIdade() {
		return idade;
	}

	public void setIdade(int idade) {
		this.idade = idade;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}
	
	public String toString() {
		return "Nome: " + nome + "\nIdade: " + idade + "\nId: " + id;
	}

}

